changelog/db.changelog-1.0.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Kirsten Williamson" id="1">
+    <tagDatabase tag="v0.0.0" />
+  </changeSet>
+  <changeSet author="Kirsten Williamson" id="2">
+    <tagDatabase tag="v1.0.0" />
+  </changeSet>
+  <changeSet author="Kirsten Williamson" id="3">
+    <createTable tableName="returns">
+      <column autoIncrement="true" name="returnsId" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="returnsId_pkey" />
+      </column>
+      <column name="schemeId" type="INTEGER" />
+      <column name="sourceSystem" type="VARCHAR(50)" />
+      <column name="paymentId" type="VARCHAR(50)" />
+      <column name="paymentJobNumber" type="VARCHAR(50)" />
+      <column name="fesCode" type="VARCHAR(50)" />
+      <column name="traderNumber" type="VARCHAR(50)" />
+      <column name="transactionNumber" type="VARCHAR(50)" />
+      <column name="invoiceNumber" type="VARCHAR(30)" />
+      <column name="sbi" type="INTEGER" />
+      <column name="frn" type="BIGINT" />
+      <column name="agreementNumber" type="VARCHAR(20)" />
+      <column name="claimNumber" type="VARCHAR(50)" />
+      <column name="currency" type="VARCHAR(3)" />
+      <column name="value" type="DECIMAL" />
```

Comment:
Would this value always be in pence at this point and therefore integer?

Reply:
Yeah I think so - will change